flask/app/routes.py

```diff
@@ -356,6 +370,34 @@ def datasets_list():
     return jsonify(datasets)
 
 
+@app.route('/api/datasets/<int:id>', methods=['GET'])
+@login_required
+def get_dataset(id: int):
+    dataset = models.Dataset.query.filter_by(dataset_id=id).options(
+        joinedload(models.Dataset.analyses),
+        joinedload(models.Dataset.tissue_sample).
+        joinedload(models.TissueSample.participant).
+        joinedload(models.Participant.family)
+     ).one_or_none()
+    if not dataset:
+        return 'Not Found', 404
```

Comment:
Could this return the ID that wasn't found or would it be exposing too much information?

Reply:
The ID is in the URL parameter `/api/datasets/123413`

Reply:
That's true, was thinking it would be more for confirmation that the ID requested was the one actually queried